src/dom/attributes/positive-boolean-attr.js

```diff
@@ -1,3 +1,24 @@
+/**
+ * ## Binding for data-f-[boolean]
+ *
+ * Flow.js provides special handling for HTML attributes that take Boolean values.
+ *
+ * In particular, for most HTML attributes that expect Boolean values, the attribute is directly set to the value of the model variable. This is true for `checked`, `selected`, `async`, `autofocus`, `autoplay`, `controls`, `defer`, `ismap`, `loop`, `multiple`, `open`, `required`, and `scoped`.
+ *
+ * However, there are a few notable exceptions. For the HTML attributes `disabled`, `hidden`, and `readonly`, the attribute is set to the *opposite* of the value of the model variable. This makes the resulting HTML easier to read.
```

Comment:
Should we js-fiddle all this when we get to breaking up the docs? Not sure, something to think about.